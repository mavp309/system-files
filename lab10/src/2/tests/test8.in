o -1
u 30
o -1
o -1
h -1
q 0
