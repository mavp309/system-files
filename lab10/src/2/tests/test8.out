Stack empty
30
Stack empty
Stack empty
